<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Magasin</class>
 <widget class="QWidget" name="Magasin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magasin</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Entrée d'un rouleau</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Diamètre : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Référence : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditReference"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditDiametre"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonNouveau">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Entrer un rouleau</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>191</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelStock">
      <property name="text">
       <string>Rouleaux dans le stock</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetStock"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonRetrait">
      <property name="text">
       <string>Sortir une série de rouleaux</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Rouleaux dans l'enceinte</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="listWidgetEnceinte"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
